common/table.js

```diff
@@ -380,8 +402,15 @@
                         return defer.promise;
                     }
 
+                    if (err instanceof ERMrest.QueryTimeoutError) {
+                        vm.countError = true;
```

Comment:
Right, this should be done in the success callbacks. I'm going to change the `hasError` properties so that the errors are `facetError`, `tableError`, `countError`, `tableError`.